concent-vm/helper-scripts/concent-reset.sh

```diff
@@ -0,0 +1,43 @@
+#! /bin/bash -e
+
+# Confirm executing script
+read -p "Are you sure about execute this script? It's destroy databases, queues and other components like: Geth etc. [y/n]" -n 1 -r
+echo
+if [[ $REPLY =~ ^[Yy]$ ]]; then
+    databases_name=(
+        concent_api
+        storage
+    )
+
+# Delete and recreate databases
+    for db_name in "${databases_name[@]}"; do
+        psql --host localhost --username postgres <<-EOF
+            DROP DATABASE IF EXISTS $db_name;
+            CREATE DATABASE $db_name;
+EOF
+    done
+
+# Migrate databases
+concent-migrate.sh
+
+# Create superuser account
+python_command="from django.contrib.auth.models import User;"
+python_command="$python_command User.objects.create_superuser('admin', 'admin@example.com', 'password')"
```

Comment:
Please use a fixture here and load it with `manage.py loaddata` (just as in the Ansible playbook: https://github.com/golemfactory/concent-deployment/pull/160/files#r205255549).